dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/AbstractTrackerPersister.java

```diff
@@ -127,23 +128,24 @@ public TrackerTypeReport persist(EntityManager entityManager, TrackerBundle bund
         //
         // Save or update the entity
         //
-        if (isNew(bundle.getPreheat(), trackerDto)) {
+        if (bundle.getStrategy(trackerDto) == TrackerImportStrategy.CREATE) {
```

Comment:
why is https://github.com/dhis2/dhis2-core/blob/112bd6fb4102b07d3a737528e48711503b3d10c7/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/TrackerImportStrategy.java#L44 not considered here? 🤔

Reply:
At this point in the importer `CREATE_AND_UPDATE` is not relevant anymore.
In [StrategyPreProcessor](https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/preprocess/StrategyPreProcessor.java) we are assigning the correct strategy to every single entity in the bundle, so here it can only be `CREATE`, `UPDATE` or `DELETE`.